src/data/sitemap_de.json

```diff
@@ -0,0 +1,22 @@
+{
+    "meta": {
+        "title": "Sitemap",
+        "description": ""
+    },
+    "section-main": {
+        "headline": {
+            "title": "Sitemap"
+        },
+        "content": {
+            "category": {
+                "homepage" : "Startseite",
+                "dashboard" : "Dashboard",
+                "faq" : "FAQ",
+                "blog" : "Blog",
+                "news": "Nachrichten",
```

Comment:
The line for "science" is missing here, so there is no header for "Science" shown on `/de/sitemap/`.

Reply:
You can mark this comment as resolved now.
Thank you!